doc/features.rst

```diff
@@ -17,7 +17,7 @@ ________
   
 * Supports a truly reproducible workflow by minimizing the code needed for analysis and visualization.
 * Already used in a variety of research projects, from conception to final publication.
-* All HoloViews objects can be pickled and unpickled.
+* All HoloViews objects can be pickled and unpickled, with no plotting-library dependencies.
```

Comment:
Don't see any reason why it wouldn't work, Palettes are just fancy cycles. There can be some issues pickling matplotlib objects (e.g. a custom ticker) but I think they were addressing that.